### PR TITLE
[12.0][ADD] slow_statement_logger

### PR DESCRIPTION
**Pull request by sbidoul**
This module lets you log slow SQL statements. It is useful when you don't have
access to the PostgreSQL log or when it is difficult to correlate the
PostgreSQL log with the Odoo log.

### CODE REVIEW

File: slow_statement_logger/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Slow SQL Statement Logger",
+    "summary": "Log slow SQL statments",
```
Comment (ThomasBinsfeld) [inline slow_statement_logger/__manifest__.py]:
```suggestion
    "summary": "Log slow SQL statements",
```

File: slow_statement_logger/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Add ``slow_statment_logger`` to Odoo's ``--load`` command line option, or to the
```
Comment (ThomasBinsfeld) [inline slow_statement_logger/readme/USAGE.rst]:
```suggestion
Add ``slow_statement_logger`` to Odoo's ``--load`` command line option, or to the
```

File: slow_statement_logger/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+``server_wide_modules`` configuration file entry.
+
+Add a ``log_min_duration_statement`` entry in the ``options`` section of your
+Odoo configuration file. Statements running at least this number of
+milliseconds will be logged with a *warning* level in the
+``odoo.addons.slow_statment_logger`` handler. ``0`` means all statements will be
```
Comment (ThomasBinsfeld) [inline slow_statement_logger/readme/USAGE.rst]:
```suggestion
``odoo.addons.slow_statement_logger`` handler. ``0`` means all statements will be
```

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by sbidoul** [comment]
After further thinking I decided to log with `debug` level.

**Review by sbejaoui — APPROVED** [review]

**Comment by sbejaoui** [comment]
Very useful, Thank you (y)

**Review by lmignon — APPROVED** [review]

**Pull request merged**